scripts/generate_embeddings.py

```diff
@@ -1,37 +1,30 @@
 #!/usr/bin/env python3
```

Comment:
should we rename this to generate_embeddings_openshift.py?

Reply:
Sounds to me :+1:

Reply:
Indeed it makes sense, I will rename it. Eventually these scripts will be moved to their own repository in openshift/lightspeed-rag-content and openstack-lightspeed/rag-content

Reply:
@umago the READMe now requires a small update as well :see_no_evil:  -> https://github.com/road-core/rag-content/blob/d43fba8741f26850c6d653378c8364b0a3d7f632/README.md?plain=1#L83

Reply:
lol ops! Indeed, done! Thank you